addons/@ocap/addons/ocap/functions/fn_handleMarkers.sqf

```diff
@@ -238,15 +258,6 @@ ocap_markers_handle = ["ocap_handleMarker", {
 				_dir = 0;
 			} else {if (_dir isEqualTo "") then {_dir = 0}};
 
-			if (["ObjectMarker", _marker] call BIS_fnc_inString) then {
-				_type = "ObjectMarker";
-				_colour = "ColorBlack";
-			};
-			if (["moduleCoverMap_dot", _marker] call BIS_fnc_inString) then {
```

Comment:
what about these?

Reply:
The icons should be included for moduleCoverMap, and recognized naturally through recording.
ObjectMarker was a special case at FNF event, but now that rectangles are accurately tracked there is no reason to continue special treatment of that marker name prefix.